src/main/java/baritone/Baritone.java

```diff
@@ -192,6 +200,11 @@ public WorldProvider getWorldProvider() {
         return worldProvider;
     }
 
+    /**
+     * TODO-yeet This shouldn't be baritone-instance specific

```

Comment:
Why's that? WorldScanner take in a world, right?

Reply:
No, it takes an ``IPlayerContext`` because it refers to the player position to determine the scan radius, and the world to scan.

Reply:
Oh whoops I'm blind, I read your comment as saying that it should be baritone instance specific. I agree that it shouldn't be.

Reply:
![Awesome](https://cdn.discordapp.com/emojis/476259277036716032.png)